CMakeLists.txt

```diff
@@ -341,22 +341,22 @@ IF( NOT LIBCP_LIB)
 ENDIF()
 INCLUDE_DIRECTORIES( ${LIBCP_INCLUDE_DIR} )
 
-SET(MPIR_PACKAGE_ROOT ${THIRD_PARTY_COMMON_DIR}/mpir)
-SET(MPIR_LIB_DIR ${MPIR_PACKAGE_ROOT}/lib)
-IF (WIN32)
-   SET(MPIR_LIB_NAME mpir)
-ELSE(WIN32)
-#   SET(MPIR_LIB_DIR ${MPIR_PACKAGE_ROOT}/lib)
-   IF(BSTERMINAL_SHARED_LIBS)
-      SET(MPIR_LIB_NAME libgmp.so)
-   ELSE()
-      SET(MPIR_LIB_NAME libgmp.a)
-   ENDIF()
-ENDIF(WIN32)
-FIND_LIBRARY(MPIR_LIB NAMES ${MPIR_LIB_NAME} PATHS ${MPIR_LIB_DIR})
-IF( NOT MPIR_LIB)
-   MESSAGE( FATAL_ERROR "Could not find MPIR lib in ${MPIR_LIB_DIR}")
-ENDIF()
+#SET(MPIR_PACKAGE_ROOT ${THIRD_PARTY_COMMON_DIR}/mpir)
+#SET(MPIR_LIB_DIR ${MPIR_PACKAGE_ROOT}/lib)
+#IF (WIN32)
+#   SET(MPIR_LIB_NAME mpir)
+#ELSE(WIN32)
+##   SET(MPIR_LIB_DIR ${MPIR_PACKAGE_ROOT}/lib)
+#   IF(BSTERMINAL_SHARED_LIBS)
+#      SET(MPIR_LIB_NAME libgmp.so)
+#   ELSE()
+#      SET(MPIR_LIB_NAME libgmp.a)
+#   ENDIF()
+#ENDIF(WIN32)
+#FIND_LIBRARY(MPIR_LIB NAMES ${MPIR_LIB_NAME} PATHS ${MPIR_LIB_DIR})
+#IF( NOT MPIR_LIB)
+#   MESSAGE( FATAL_ERROR "Could not find MPIR lib in ${MPIR_LIB_DIR}")
+#ENDIF()
```

Comment:
We still need this on Windows, so I'd better remove only the `ELSE(WIN32)` part (and put the rest of MPIR detection under `IF(WIN32)`.